src/extension.ts

```diff
@@ -338,6 +338,9 @@ export function activate(context: VSCode.ExtensionContext) {
   context.subscriptions.push(
     VSCode.commands.registerCommand(Commands.CONNECT_TO_SONARQUBE, connectToSonarQube(context))
   );
+  context.subscriptions.push(
+      VSCode.commands.registerCommand(Commands.MODIFY_CONNECTION, editConnection(context))
```

Comment:
Minor thing but maybe it would be better to use uniform terminology, either `edit` or `modify` or `update`